bee-common/src/logger/mod.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+mod config;
+
+pub use config::{LoggerConfig, LoggerConfigBuilder, LoggerOutputConfig, LoggerOutputConfigBuilder};
+
+use fern::{
+    colors::{Color, ColoredLevelConfig},
+    Dispatch,
+};
+
+/// Name of the standard output.
+pub const LOGGER_STDOUT_NAME: &str = "stdout";
+
+/// Error occuring when initializing a logger backend.
+#[derive(Debug)]
+#[non_exhaustive]
+pub enum Error {
```

Comment:
If this type is part of the public API, implementing `std::error::Error` for it might be a good idea too. Most error libraries that box errors do to by boxing `dyn Error`, not `dyn Any`.